espeak-ng-data/lang/sem/ar

```diff
@@ -1,21 +1,21 @@
 name Arabic
 language ar
 phonemes ar
-pitch 160 120
-roughness 0
-speed 70
-
-formant 0 105 80 150 
-formant 1 110 80 160 
-formant 2 110 70 150 
-formant 3 110 70 150 
-formant 4 115 80 200 
-formant 5 115 80 100 
-formant 6 110 70 150 
-formant 7 110 70 100 
-formant 8 110 70 150 
-
-stressAdd 0 0 -10 -10 0 0 10 40 
-echo 20 10 
-voicing 75 
-consonants 150 150
+//pitch 160 120
+//roughness 0
+//speed 70
+//
+//formant 0 105 80 150 
+//formant 1 110 80 160 
+//formant 2 110 70 150 
+//formant 3 110 70 150 
+//formant 4 115 80 200 
+//formant 5 115 80 100 
+//formant 6 110 70 150 
+//formant 7 110 70 100 
+//formant 8 110 70 150 
+//
+//stressAdd 0 0 -10 -10 0 0 10 40 
+//echo 20 10 
+//voicing 75 
+//consonants 150 150
```

Comment:
Same as with the previous commit -- delete removed lines. It keeps the files clean.